### PR TITLE
fix bug that fisco-bcos adapter can't open network certificate file

### PR DESCRIPTION
**Pull request by vita-dounai**

### CODE REVIEW

**Comment by vita-dounai** [comment]
@aklenik Hi, could you have a review for this PR? Thanks!

File: packages/caliper-fisco-bcos/lib/fiscoBcos.js [hunk 1]
```diff
@@ -38,6 +39,12 @@ class FiscoBcos extends BlockchainInterface {
         this.bcType = 'fisco-bcos';
         this.workspaceRoot = workspace_root;
         this.fiscoBcosSettings = CaliperUtils.parseYaml(this.configPath)['fisco-bcos'];
+
+        if (this.fiscoBcosSettings.network && this.fiscoBcosSettings.network.authentication) {
+            for (let k in this.fiscoBcosSettings.network.authentication) {
+                this.fiscoBcosSettings.network.authentication[k] = path.join(this.workspaceRoot, this.fiscoBcosSettings.network.authentication[k]);
```
Comment (aklenik) [inline packages/caliper-fisco-bcos/lib/fiscoBcos.js]: For easily/correctly resolving both absolute and relative paths, use `CaliperUtils.resolvePath(<path>)`:
https://github.com/hyperledger/caliper/blob/master/packages/caliper-core/lib/common/utils/caliper-utils.js#L67

Comment (vita-dounai) [reply packages/caliper-fisco-bcos/lib/fiscoBcos.js]: You are right, I've modified it.

Comment (vita-dounai) [reply packages/caliper-fisco-bcos/lib/fiscoBcos.js]: @aklenik

**Pull request merged**